src/styles/components/_syntax-highlighter.scss

```diff
@@ -135,23 +136,17 @@ pre > code.diff-highlight .token.inserted:not(.prefix) {
             font-size: var(--mynah-syntax-code-font-size) !important;
             font-family: var(--mynah-syntax-code-font-family) !important;
         }
+
         code {
             color: var(--mynah-color-syntax-code);
             filter: brightness(0.95);
-            white-space: pre;
             background-color: inherit;
+            white-space: pre-wrap;
+            word-wrap: break-word;
+            overflow-wrap: anywhere;
+            word-break: break-all;
         }
```

Comment:
Do I correctly understand that with this change we'll get all the code blocks wrapped, so that there will be no horizontal scrolling for the code blocks at all? This is a considerable change and we need to double-check that this will provide better use experience.